packages/typespec-azure-resource-manager/lib/common-types/extended-location-ref.tsp

```diff
@@ -0,0 +1 @@
+import "./extended-location.tsp";
```

Comment:
Why is this here?

Reply:
I just left the `xxx.tsp` and `xxx-ref.tsp` pattern. Once we start generating the common types swagger, we will need to add decorator here.